apps/mochi-web/components/auth-layout.tsx

```diff
@@ -66,44 +201,66 @@ export default function AuthenticatedLayout({
       {!isLoadingSession && isLoggedIn ? (
         <Layout className="flex-1">
           <Sidebar
-            Header={SidebarHeader}
-            headerItems={[
-              {
-                title: 'Profile',
-                Icon: UserSolid,
-                type: 'link',
-                as: Link,
-                href: ROUTES.MY_PROFILE,
-              },
-              { title: 'Servers', Icon: Discord },
-              { title: 'Settings', Icon: SettingSolid },
-              {
-                title: 'Developer',
-                Icon: CodingSolid,
-                type: 'link',
-                as: Link,
-                href: ROUTES.APPLICATON_LIST,
-                badge: getSidebarBadge['NEW'],
-              },
-              { title: 'Invite Friends', Icon: AddUserSolid },
-            ]}
-            footerItems={[
-              { title: 'Support', Icon: LifeBuoySolid },
-              {
-                title: 'Follow Us',
-                Icon: X,
-                type: 'link',
-                href: TWITTER_LINK,
-              },
-              {
-                title: 'Join Community',
-                Icon: Discord,
-                type: 'link',
-                href: DISCORD_LINK,
-              },
-            ]}
+            Header={
+              variant === 'main'
+                ? MainSidebarHeader
+                : ApplicationDetailSidebarHeader
+            }
+            headerItems={
+              variant === 'main'
+                ? [
+                    {
+                      title: 'Profile',
+                      Icon: UserSolid,
+                      type: 'link',
+                      as: Link,
+                      href: ROUTES.MY_PROFILE,
+                    },
+                    { title: 'Servers', Icon: Discord },
+                    { title: 'Settings', Icon: SettingSolid },
+                    {
+                      title: 'Developer',
+                      Icon: CodingSolid,
+                      type: 'link',
+                      as: Link,
+                      href: ROUTES.APPLICATON_LIST,
+                      badge: getSidebarBadge['NEW'],
+                    },
+                    { title: 'Invite Friends', Icon: AddUserSolid },
+                  ]
+                : [
+                    {
+                      title: 'Overview',
+                      Icon: HomeSolid,
+                      pattern: APPLICATION_DETAIL_ROUTE_REGEX,
+                    },
+                    { title: 'Revenue', Icon: SafeBoxSolid },
+                  ]
+            }
```

Comment:
Maybe consider a clearer way to include the data based on a variant. For example:

```
const content = {
   'main': {Header: ..., headerItems: ...},
   'app-detail: {...}
}

Header={content[variant].header}
headerItems={content[variant].headerItems}
```